tests/functional/sca/test_sca_scan_diff.py

```diff
@@ -0,0 +1,50 @@
+from pathlib import Path
+
+from tests.functional.utils import run_ggshield_sca_scan
+from tests.repository import Repository
+
+
+def test_scan_ci_diff(tmp_path: Path, monkeypatch, pipfile_lock_with_vuln) -> None:
```

Comment:
```suggestion
def test_scan_diff(tmp_path: Path, monkeypatch, pipfile_lock_with_vuln) -> None:
```

Reply:
and it should work without `monkeypatch`

Reply:
That is what happens when you copy paste a bit too fast... Sorry for that. I fixed it.